chapter01/Intro_example_short.ipynb

```diff
@@ -0,0 +1,604 @@
+    "\n",
+    "For this problem, all the points $(x_1, x_2)$ that satisfy $x_1 + x_2 = b^\\mathrm{obs}$ are solutions to this maximization problem. These solutions are depicted as the red line in the likelihood plot below."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Plot the likelihood\n",
+    "plot_2D_density(\n",
+    "    likelihood,\n",
+    "    v1_min=x1_lim[0], v1_max=x1_lim[1],\n",
+    "    v2_min=x2_lim[0], v2_max=x2_lim[1])\n",
+    "\n",
+    "# Plot the line x2 = b_obs - x1\n",
+    "plt.plot(x1_lim, b_obs-x1_lim, '--r')\n",
+    "plt.ylim(x2_lim);"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Combining the likelihood with the prior will give us a unique maximum a posteriori (MAP) point estimate as we will see next."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### <font color=#8B4513> 1.6. Putting it all together, the BIP </font> <a class=\"anchor\" id=\"r-putting-it-all-together-the-bip\"></a>\n",
+    "\n",
+    "#### Posterior definition using the Bayes’ rule\n",
+    "The posterior is proportional to the product of **likelihood** and **prior**\n",
+    "\n",
+    "$$\\pi(\\mathbf{x} | b=b^\\mathrm{obs}) \\propto L (\\mathbf{x} | b=b^\\mathrm{obs}) \\pi(\\mathbf{x})$$\n",
+    "\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In CUQIpy, we can use the class `BayesianProblem` to bundle the prior, the data distribution, and the data, then use it to explore the posterior distribution (e.g. point estimate and sampling):"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "BP = BayesianProblem(b, x)\n",
+    "print(BP)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we pass the data:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "BP.set_data(b=b_obs)\n",
+    "print(BP)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Note the difference in the target of the `BayesianProblem` object before and after passing the data."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## <font color=#CD853F> 2. Solving the BIP </font> <a class=\"anchor\" id=\"r-solving-the-bip\"></a>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### <font color=#8B4513> 2.1. Maximum a posteriori (MAP) estimate </font> <a class=\"anchor\" id=\"r-maximum-a-posteriori-map-estimate\"></a>\n",
+    " \n",
+    "Maximizer of the posterior\n",
+    "\n",
+    "$$\\mathbf{x}^* = \\underset{\\mathbf{x}}{\\operatorname{argmax\\;}} \\pi(\\mathbf{x} | b=b^\\mathrm{obs})$$\n",
```

Comment:
There seems a jump between this equation and the one below? What do you think? For example, from maximization to minimization.

Reply:
Thanks for the MS teams chat on this, resolved based on that chat.